integrations/server/test_covidcast_endpoints.py

```diff
@@ -164,6 +269,64 @@ def test_basic(self):
             out = self._fetch("/", signal=first.signal_pair, geo=first.geo_pair, time="day:*")
             self.assertEqual(len(out["epidata"]), len(rows))
 
+        rows = [CovidcastRow(source="jhu-csse", signal="confirmed_cumulative_num", time_value=20200401 + i, value=i) for i in range(10)]
+        self._insert_rows(rows)
+        rows = [CovidcastRow(source="jhu-csse", signal="confirmed_cumulative_num", time_value=20200401 + i, value=2*i, geo_value="01235") for i in range(10)]
+        self._insert_rows(rows)
+
+        rows = [CovidcastRow(source="jhu-csse", signal="confirmed_unknown", time_value=20200401 + i, value=i) for i in range(10)]
+        first = rows[0]
+        self._insert_rows(rows)
+
+        with self.subTest("unknown signal"):
+            out = self._fetch("/", signal="jhu-csse:confirmed_unknown", geo=first.geo_pair, time="day:*")
+            self.assertEqual(len(out["epidata"]), len(rows))
+            out_values = [row["value"] for row in out["epidata"]]
+            expected_values = [float(i) for i in range(10)]
```

Comment:
why not using the rows object and their values directly?

Reply:
Do you mean to compute the expected_values via differences?

Reply:
I mean when you were creating the database entries you were using a list of CovidcastEntry objects. You could use these object also to verify the output. Thus, you don't have to use internal knowledge (like the range(10) thing) to verify the output

Reply:
got it